guava-tests/test/com/google/common/hash/BloomFilterTest.java

```diff
@@ -255,15 +255,16 @@ public void testNullPointers() {
   /** Tests that we never get an optimal hashes number of zero. */
   public void testOptimalHashes() {
     for (int n = 1; n < 1000; n++) {
-      for (int m = 0; m < 1000; m++) {
-        assertTrue(BloomFilter.optimalNumOfHashFunctions(n, m) > 0);
+      for (double p = 0.1; p > 1e-10; p/=10) {
+        assertTrue(BloomFilter.optimalNumOfHashFunctions(p) > 0);
       }
     }
   }
 
   // https://github.com/google/guava/issues/1781
   public void testOptimalNumOfHashFunctionsRounding() {
```

Comment:
Should this name be changed?

I realize that the existing test doesn't do this, but I think it would be good to spell out the calculation that goes from 0.03 to 5 in a comment.